src/client/mod.rs

```diff
@@ -118,12 +131,16 @@ impl std::fmt::Debug for ClientBuilder {
 #[derive(Debug)]
 pub struct Client {
     brokers: Arc<BrokerConnector>,
+    backoff_config: Option<BackoffConfig>,
```

Comment:
Could we just make this non-optional? It seems like you are always creating it by default anyway, might as well just create it when the client is being instanced instead of each time a subcomponent is.

Reply:
I agree